.github/workflows/e2e.yaml

```diff
@@ -8,6 +8,8 @@ on:
 
 jobs:
   e2eTests:
+    # Do not run e2e tests if commit message or PR has skip-e2e.
+    if: ! ( contains(github.event.head_commit.message, 'skip-e2e') || contains(github.event.pull_request.title, 'skip-e2e') )
```

Comment:
There is a way to do this with labels as well I think, but I'm not sure if we prefer that. What do you think?

Reply:
It would be enough with title and commits, but also labels would be nice with the opt in/out feature.

https://stackoverflow.com/questions/62325286/run-github-actions-when-pull-requests-have-a-specific-label

the others are a bit more "static"

Reply:
Switched to `github.event.pull_request.labels.*.name, 'ci/skip-e2e'`, seems to be working fine :)

I gave a thought to reacting on the `unlabeled` event (to see if someone removes the `ci/skip-e2e` label), but I think that might backfire on us causing the e2e to be re-run every time anybody changes the labels on the PR for any other reason, as I haven't found a way to react to the event only if that label is being removed :(